x-pack/legacy/plugins/reporting/server/browsers/chromium/driver/chromium_driver.ts

```diff
@@ -45,8 +50,18 @@ export class HeadlessChromiumDriver {
     logger.info(`opening url ${url}`);
     await this.page.setRequestInterception(true);
     let interceptedCount = 0;
+
```

Comment:
Even though we allow users to configure allow/deny patterns for tighter control, I opted to _always_ fail `file://` requests due to their threat level. If a user _wants_ to serve stuff from their file system they should put it behind a webserver, otherwise doing this is just a huge security hole